### PR TITLE
implemented booking model routes and controller

### PR DESCRIPTION
**Pull request by Corentr1**
Hey guys!

I've just implemented the bookings model. The migration file is ready and works on my end.

Could you please check my validations and my routes? (not sure if the routes are good)

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -2,5 +2,8 @@
   devise_for :users
   root to: 'pages#home'
   # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
-  resources :costumes
+  resources :costumes do
+    resources :bookings, only: [:new]
+  end
+  resources :bookings, only: %i[index create edit destroy]
```
Comment (11012022) [inline config/routes.rb:8]: Why did you put the index, create, edit here and not above ?

**Comment by 11012022** [comment]
Looks good to me

File: config/routes.rb (hunk 1 above)
Comment (Corentr1) [reply config/routes.rb:8]: Only the new was nested in our routes. The rest shouldn't 😉

**Comment by JonnySomi** [comment]
Looks good to me!

**Pull request merged**